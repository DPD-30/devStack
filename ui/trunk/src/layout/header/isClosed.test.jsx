import React from 'react'
import { renderToString } from 'react-dom/server'


import LayoutHeaderIsClosed from './isClosed.jsx'

jest.unmock('./isClosed.jsx')


test('LayoutHeaderIsClosed matches snapshot', () => {
	const comp = renderToString(<LayoutHeaderIsClosed />)

	expect(comp).toMatchSnapshot()
})
