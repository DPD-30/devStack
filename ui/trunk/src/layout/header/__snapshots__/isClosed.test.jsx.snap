// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LayoutHeaderIsClosed matches snapshot 1`] = `
<div class="header"
     data-reactroot
>
  <div class="container-fluid">
    <div class="row">
      <div class="appName col-8">
        CACI Challenge App
      </div>
      <div class="userName col-4">
      </div>
    </div>
  </div>
</div>
`;
